### ISSUE TITLE
Drop Python 2

### ISSUE THREAD

**Issue by craigemery** [craigemery/vim-autotag#28]
I have all this code to support Python 2.4 etc and I'm quite of the opinion that I should only support Python 3.

**Comment by eupharis** [comment]
@craigemery here here!

Just FYI dropping Python2 would fix a bug I was having using your library. When I upgraded to vim8, your (awesome) library broke and I finally just got around to figuring out why. Turns out my vim8 has python3 support but no python2.

![image](https://user-images.githubusercontent.com/847862/77698292-9b8c4100-6f6d-11ea-9c2d-835c2bbba2b5.png)

I just hacked at the autotag.vim locally to prefer python3 and now it works again for me. But there could be others in the same boat and dropping python2 support would fix it for them.
